server.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+
+const app = express();
+const PORT = 3000;
+
+app.use(express.static('./dist'));
+
+app.listen(PORT, function () {
```

Comment:
Для анонимных функций обычно используют, стрелочные функции
```suggestion
app.listen(PORT, () => {
```

Reply:
Я на самом деле этот код просто из главы с express скопировал. Но в целом да, согласен.